linebot/insight/api_insight.go

```diff
@@ -217,12 +257,36 @@ func (client *InsightAPI) GetNumberOfFollowers(
 	date string,
 
 ) (*GetNumberOfFollowersResponse, error) {
+	response, body, error := client.GetNumberOfFollowersWithHttpInfo(
+
+		date,
+	)
+
+	defer response.Body.Close()
```

Comment:
is there any problem calling `defer response.Body.Close()` twice ...? `~WithHttpInfo` calls `defer res.Body.Close()` and original methods call `defer response.Body.Close()` again.